templates/data/lifecycle-manager-data.yaml

```diff
@@ -37,42 +61,34 @@ templates:
                     - "jobConfig_presubmit"
                     - "build_labels" # default labels
               - jobConfig:
-                  labels:
-                    preset-docker-push-repository-kyma: "true" # sets GCR Registry through DOCKER_PUSH_REPOSITORY
-                  name: pull-build # "pre-main-lifecycle-manager-build"
+                  name: pull-build
                   always_run: true
-                  command: "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build-generic.sh"
                   args:
-                    - "/home/prow/go/src/github.com/kyma-project/lifecycle-manager/operator"
-                    - "docker-build"
-                    - "docker-push"
-                  branches:
-                    - ^main$
+                    - "--name=lifecycle-manager"
+                    - "--config=/config/kaniko-build-config.yaml"
+                    - "--context=operator"
+                    - "--dockerfile=Dockerfile"
+                    - "--directory=."
                 inheritedConfigs:
+                  local:
+                    - default
                   global:
                     - jobConfig_default
                     - jobConfig_presubmit
-                    - extra_refs_test-infra
-                    - image_buildpack-golang
-                    - jobConfig_generic_component
                     - "build_labels" # default labels
```

Comment:
please remove jobConfig_default and build-labels from there. These sets include set up for privileged mode and build labels that enable docker-in-docker. You don't need them in kaniko-build